### ISSUE TITLE
Numbers are retrieved incorrectly

### ISSUE THREAD

**Issue by Emtschikay** [NathanaelA/nativescript-sqlite#59]
Hey,

I have a numeric column in the table "member" of my SQLite db "test". It has a Integer column "external_id", which holds a value of 102579790437329. If I do 
```
new Sqlite("test.db")
                .then((db) => {
                    db.resultType(Sqlite.RESULTSASOBJECT);
                    return db.get(
                        "SELECT external_id FROM member"
                    );
                })
                .then((result) => {
                    console.log(result.external_id);
                    resolve(result);
                })
                .catch((err) => {
                    reject(err);
                });
```

I get -1208460335. If I change the column type to Text, everything works as expected. Also, if I lower the value to 10000 for example, I also retrieve the right value. Any idea why that is? Am I hitting some max value for SQLite (which should be 2^63-1 from what I've read) or JavaScript (which should be 9007199254740991)?

Thanks in advance

**Comment by NathanaelA** [comment]
Is this happening on Android or iOS?

**Comment by Emtschikay** [comment]
I unfortunately can't test it right now, but I'm pretty sure it was on iOS

**Comment by NathanaelA** [comment]
Ok, then I know exactly what the issue is.  I'll get it fixed later this week.  Getting ready for the NS Dev Days conf tomorrow...

**Comment by Emtschikay** [comment]
Cool, thanks a ton for looking into it. Would also be interested in the cause of the problem :)

**Comment by jacargentina** [comment]
@NathanaelA i have a case where I store on a REAL Column the value `6186.20`, and when doing a select on that field I get `6186.2001953125`... Is it related to this ticket too? Thanks!

**Comment by NathanaelA** [comment]
@jacargentina - Your issue is related to #58 ; this issue is caused by the getInt call in iOS, it needs to be a long call.

**Comment by ZixiaoWang** [comment]
I've met the very similar issue here, on iOS devices.
I tried to retrieve a javascript timestamp from database but I got a minus number instead.
Like the timestamp I saved is 1506960000000 but the number I retrieved is -573520896.
And it seems like the maximum size of it is 1507533520896.

So could you please enlighten me where should I look at and how I could possibly resolve the issue. Thanks!

**Comment by NathanaelA** [comment]
This should be fixed in v1.1.10.
